guides/process/scheduling/how_to_timesheet.md

```diff
@@ -44,6 +44,20 @@ You should allocate your time to a project when you’ve spent 2 hours or more.
 
 We use Kimble to complete timesheets. All resources on how to complete and approve timesheets are in the [Knowledge base](https://sites.google.com/madetech.com/signpost/home/kimble-resources?authuser=0).
 
+### Why we need you to do timesheets
+
+Timesheet data directly drives our client invoicing. If you don't timesheet, the business can't invoice and we lose revenue which has an impact on all of us.
```

Comment:
I don't love the wording of this and it's not strictly accurate. Would suggest changing it to something like
'Timesheet data directly drives our client invoicing. Incomplete and incorrect timesheets can lead to us under or overcharging clients which can create significant challenges for account and finance teams.'

Reply:
Updated